lib/hooks/use-error-codes.ts

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { useReferenceData } from '../api/reference-data/v1';
+import locale from '../locale';
+
+export const useErrorCodes = () => {
+  const [errors, setErrors] = useState<any>(locale.hooks.errorMessages);
```

Comment:
Can you change this to `errorMessages`, I was a little confused to what was an error form the api and what the messages were.

Reply:
Good shout, yep can change